src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_FOR_COLUMNS = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final String BUY_TYPE = "buy";
+    private static final String SUPPLY_TYPE = "supply";
+    private static final String RESULT_TYPE = "result";
+    private static final String WRITING_TO_FILE_ERROR = "Sorry, but i can't write in file";
```

Comment:
Don't forget about spaces to avoid concatenation with following words
```suggestion
    private static final String WRITING_TO_FILE_ERROR = "Sorry, but i can't write in file ";
```